app/Livewire/Project/Application/General.php

```diff
@@ -166,25 +119,25 @@ protected function messages(): array
         'application.dockerfile' => 'Dockerfile',
         'application.docker_registry_image_name' => 'Docker registry image name',
         'application.docker_registry_image_tag' => 'Docker registry image tag',
+        'application.docker_use_custom_registry' => 'Use private registry',
         'application.dockerfile_location' => 'Dockerfile location',
         'application.docker_compose_location' => 'Docker compose location',
         'application.docker_compose' => 'Docker compose',
         'application.docker_compose_raw' => 'Docker compose raw',
         'application.custom_labels' => 'Custom labels',
         'application.dockerfile_target_build' => 'Dockerfile target build',
         'application.custom_docker_run_options' => 'Custom docker run commands',
-        'application.custom_network_aliases' => 'Custom docker network aliases',
         'application.docker_compose_custom_start_command' => 'Docker compose custom start command',
         'application.docker_compose_custom_build_command' => 'Docker compose custom build command',
         'application.custom_nginx_configuration' => 'Custom Nginx configuration',
         'application.settings.is_static' => 'Is static',
-        'application.settings.is_spa' => 'Is SPA',
         'application.settings.is_build_server_enabled' => 'Is build server enabled',
         'application.settings.is_container_label_escape_enabled' => 'Is container label escape enabled',
         'application.settings.is_container_label_readonly_enabled' => 'Is container label readonly',
         'application.settings.is_preserve_repository_enabled' => 'Is preserve repository enabled',
         'application.watch_paths' => 'Watch paths',
         'application.redirect' => 'Redirect',
+        'selectedRegistries' => 'Registries',
```

Comment:
Why would someone get one image (in theory same image) from more than one registry? What is the point in doing it? Never saw it before...